src/main/java/com/annapolisworks/Operator.java

```diff
@@ -0,0 +1,48 @@
+package com.annapolisworks;
+
+public enum Operator {
+    PLUS('+', 1),
+    MINUS('-', 1),
+    MULTIPLY('*', 2),
+    DIVIDE('/', 2),
+    EXPONENT('^', 3);
+
+    private char mOper;
+    private int mPriority;
+
+    Operator(char operator, int priority) {
```

Comment:
you mean the constructor? Intellij didn't like it because it says enums always have private constructors.